google-cloud-bigtable/src/main/resources/changestream.json

```diff
@@ -0,0 +1,1232 @@
+    },
+    {
+      "description": "1 unchunked SetCell + 1 chunked SetCell([{SC1, SC2_chunk1}, {SC2_chunk2}]->ChangeStreamMutation{SC1,SC2})",
+      "api_responses": [
+        {
+          "data_change": {
+            "row_key": "0000000000000000",
+            "type": "USER",
+            "source_cluster_id": "source-cluster-id",
+            "commit_timestamp": "2022-07-01T00:00:00Z",
+            "tiebreaker": 100,
+            "chunks": [
+              {
+                "mutation": {
+                  "set_cell": {
+                    "family_name": "family",
+                    "column_qualifier": "0000000000000000",
+                    "timestamp_micros": 1000,
+                    "value": "dg=="
+                  }
+                }
+              },
+              {
+                "chunk_info": {
+                  "chunked_value_size": 1
+                },
+                "mutation": {
+                  "set_cell": {
+                    "family_name": "family",
+                    "column_qualifier": "0000000000000000",
+                    "timestamp_micros": 1000,
+                    "value": "dg=="
+                  }
+                }
+              }
+            ]
+          }
+        },
+        {
+          "data_change": {
+            "type": "CONTINUATION",
+            "token": "data-change-token",
+            "low_watermark": "2022-07-01T00:00:00Z",
+            "chunks": [
+              {
+                "chunk_info": {
+                  "chunked_value_offset": 1,
+                  "chunked_value_size": 1,
+                  "last_chunk": true
+                },
+                "mutation": {
+                  "set_cell": {
+                    "value": ""
+                  }
+                }
+              }
+            ],
+            "done": true
+          }
+        }
+      ],
+      "results": [
+        {
+          "record": {
+            "change_stream_mutation": {
+              "row_key": "0000000000000000",
+              "type": "USER",
+              "source_cluster_id": "source-cluster-id",
+              "commit_timestamp": "2022-07-01T00:00:00Z",
+              "tiebreaker": 100,
+              "token": "data-change-token",
+              "low_watermark": "2022-07-01T00:00:00Z",
+              "mutations": [
+                {
+                  "set_cell": {
+                    "family_name": "family",
+                    "column_qualifier": "0000000000000000",
+                    "timestamp_micros": 1000,
+                    "value": "dg=="
+                  }
+                },
+                {
+                  "set_cell": {
+                    "family_name": "family",
+                    "column_qualifier": "0000000000000000",
+                    "timestamp_micros": 1000,
+                    "value": "dg=="
+                  }
+                }
+              ]
+            }
+          },
+          "error": false
+        }
+      ]
+    },
```

Comment:
maybe also add a test for {SC_CHUNK1}, {SC_CHUNK2}, {SC_CHUNK3} ?

Reply:
Done.